packages/amplication-server/prisma/migrations/20220221165146_21022022_change_enum_type/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
```

Comment:
I recommend deleting all the migrations and create a single migration for all the changes in this PR - unless there is a real need for stepped migration